docs/source/getting_started/platform.rst

```diff
@@ -159,12 +159,38 @@ drag-and-drop or using the Python Package according to:
     You can upload up to 1'000 images using the frontend.
 
 
+Images can also be uploaded from a Python script:
+
+.. code-block:: python
+
+    from lightly.api.api_workflow_client import ApiWorkflowClient
+    client = ApiWorkflowClient(token='123'm dataset_id='xyz')
+
+    # change mode to thumbnails or meta if you're working with sensitive data
```

Comment:
would put thumbnails and meta in quotes (e..g 'thumbnails', 'meta') to make it clear